src/Illuminate/Http/Client/PendingRequest.php

```diff
@@ -715,7 +736,7 @@ public function post(string $url, $data = [])
      * @param  array  $data
      * @return \Illuminate\Http\Client\Response
      */
-    public function patch($url, $data = [])
+    public function patch(string $url, $data = [])
```

Comment:
The other methods were already type-hinted, so I updated these too. Let me know if this is wrong though.